Python/fileutils.c

```diff
@@ -2019,7 +2036,12 @@ _Py_wrealpath(const wchar_t *path,
 int
 _Py_isabs(const wchar_t *path)
 {
-#ifdef MS_WINDOWS
+#ifdef MS_WINDOWS_GAMES
+    /* this does not handle persistent local storage */
+    return (wcsncmp(path, L"G:\\", 3) == 0)
+        || (wcsncmp(path, L"D:\\", 3) == 0)
+        || (wcsncmp(path, L"T:\\", 3) == 0);
```

Comment:
```suggestion
    if (path[0] == L'\\' || path[0] == L'/') {
        // Check for an absolute UNC path.
        return path[1] == L'\\' || path[1] == L'/';
    }
    else {
        // Check for an absolute drive path.
        return ((path[0]) &&
                (path[1] == L':') &&
                (path[2] == L'\\' || path[2] == L'/'));
    }
```
This is generalized to support using forward slashes, any drive-letter drive "[A-Za-z]:", and UNC paths. Even if UNC paths and configuring arbitrary drives aren't supported currently on Xbox, they could be at some point and should be supported in principle, as valid Windows paths.